stores/messagesStore/controls.js

```diff
@@ -13,3 +13,42 @@ export async function REQUEST_THREADS(action) {
   const response = await fetch(url);
   return await response.json();
 }
+
+export async function REQUEST_MESSAGE_LIKE(action) {
+  await fetch('/api/messages?action=messageLike', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      fromUserId: action.fromUserId,
+      index: action.index,
+      like: action.like,
+    }),
+  });
+}
+
+export async function DELETE_MESSAGE(action) {
+  await fetch('/api/messages?action=deleteMessage', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      fromUserId: action.fromUserId,
+      index: action.index,
+    }),
+  });
+}
+export async function SUBMIT_MESSAGE(action) {
+  await fetch('/api/messages?action=submitMessage', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      fromUserId: action.fromUserId,
+      text: action.text,
+    }),
+  });
+}
```

Comment:
I can't explain this. I don't know how this happened.

Reply:
Me either